models/issue_comment.go

```diff
@@ -110,6 +110,10 @@ const (
 	CommentTypeDismissReview
 	// 33 Change issue ref
 	CommentTypeChangeIssueRef
+	// 34 pr was scheduled to auto merge when checks succeed
+	CommentTypePRScheduledToAutoMerge
+	// 35 pr was un scheduled to auto merge when checks succeed
+	CommentTypePRUnScheduledToAutoMerge
```

Comment:
New comment types must be added to the string list below https://github.com/go-gitea/gitea/blob/5d3f346af4fd5703fe1044e6649f654aff1a1cb4/models/issue_comment.go#L153